### PR TITLE
added Add feature

### PR DESCRIPTION
**Pull request by ChengJiyuqing**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -133,6 +133,25 @@ Classes used by multiple components are in the `seedu.medmoriser.commons` packag
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Feature (Cheng Jiyuqing)
+
+### Implementation
```
Comment (joshruien) [inline docs/DeveloperGuide.md]: should be 4 #

Comment (ChengJiyuqing) [reply docs/DeveloperGuide.md]: okay changed!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -133,6 +133,25 @@ Classes used by multiple components are in the `seedu.medmoriser.commons` packag
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Feature (Cheng Jiyuqing)
+
+#### Implementation
+
+This portion describes the implementation of the add feature which allows the user to add new QAndA sets with the question, answer and/or tags.
```
Comment (jonfoocy) [inline docs/DeveloperGuide.md]: I think "and/or" is wrong, because it means answer or tags. I think can change to question, answer and tags (optional).

Comment (jonfoocy) [reply docs/DeveloperGuide.md]: Also I think can remove the 'sets' and just leave it as "add new QAndAs"

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -133,6 +133,25 @@ Classes used by multiple components are in the `seedu.medmoriser.commons` packag
...
+
+Explanation of the diagrams:
+
+1. As the user parses the add command through the `UI`, the `MedmoriserParser` identifies this as a `AddCommand` and passes the user input to the `AddCommandParser`.
+2. The `AddCommandParser` ensures the input is of correct format and identifies the input for question, answer and tag(s).
+3. The `AddCommandParser` then creates a new QAndA set object, and passes it to a new `AddCommand` object to handle the rest of the job.
```
Comment (jonfoocy) [inline docs/DeveloperGuide.md]: Can remove 'set'

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 ### Architecture
 
-<img src="images/ArchitectureDiagram.png" width="450" />
+<img src="images/ArchetectureDiagram.png" width="450" />
```
Comment (jonfoocy) [inline docs/DeveloperGuide.md]: typo - should be ArchitectureDiagram. Remember to change your image filename too!

**Review by jonfoocy — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (ChengJiyuqing) [reply docs/DeveloperGuide.md]: okay changeed!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ChengJiyuqing) [reply docs/DeveloperGuide.md]: changed! thanks!

**Review by yongmingyang — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (ChengJiyuqing) [reply docs/DeveloperGuide.md]: changed!

**Review by jonfoocy — APPROVED** [review]

**Pull request merged**